dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/layers/PoolingLayerTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2015 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.layers;
+
+import edu.snu.dolphin.dnn.blas.Matrix;
+import edu.snu.dolphin.dnn.blas.MatrixFactory;
+import edu.snu.dolphin.dnn.blas.jblas.MatrixJBLASFactory;
+import edu.snu.dolphin.dnn.conf.*;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.Tang;
+import org.apache.reef.tang.exceptions.InjectionException;
+import org.junit.Before;
+import org.junit.Test;
+import edu.snu.dolphin.dnn.conf.LayerConfigurationParameters.*;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test class for pooling layer.
+ */
```

Comment:
We should have add `@FixMethodOrder(MethodSorters.NAME_ASCENDING)` for test success.
`testMaxPoolingActivation()` should be executed before `testMaxPoolingBackPropagate()` is executed. Here is the link about JUnit's test execution order [link](https://github.com/junit-team/junit/wiki/Test-execution-order).